packages/replay-web/src/device.ts

```diff
@@ -46,6 +47,7 @@ export function getAudio(
       );
     }
     const { buffer, playState } = data;
+    const gainNode = playState?.gainNode || audioContext.createGain();
```

Comment:
I'm worried that the gainNode may not get cleaned up or conflict with other sounds. How about you create a new one every time `play` is called, and then access it in `setVolume` etc through `playState`?

Reply:
Done!